README.md

```diff
@@ -78,6 +78,24 @@ storage:
   # configure where to store keys and certificates; described later
 challenge_responders:
   # configure how to respond ACME challenges; described later
+  - route53: {}
+
+  # configure your customize challenge domain(AWS route53) here
```

Comment:
Wrong location to add the doc. Didn't you see "described later" in the section you're adding to?